egs/wsj/s5/steps/dur_model/get_examples.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2015 Hossein Hadian
```

Comment:
I suggest renaming this script to dur_model/get_egs.sh. Add a comment describing what this file does. e.g. see the description in steps/nnet3/get_egs.sh